README.md

```diff
@@ -140,6 +140,27 @@ Follow these steps:
 - After successful deployment you will see the API URLs. Copy the URLs that ends on `/initialization`, post them to
   the browser URL input and click enter. They will create all needed stuff in your iconik account.
 
+5. iconik Security Workflow - allows to rotate your tokens
+
+For admins:
+
+- After initializing all workflows, copy the URL that ends on `/api/security/initialization`,
+  post it to the browser URL input and click enter. It will create all
+  needed stuff in your iconik account.
+- You need to use CA `Change Refresh Token Period`, which starts iconik Security
```

Comment:
Could you explay why the service starts work after calling this CA? Change Refresh Token Period - it says "change refresh token period", but as a user i don`t expect this CA do something else than changing refresh period. What about seting default refresh period during the initialization?

Reply:
if the user has this feature, then we will first enable it ourselves and he can only change the time

@azhivaeva-flo what do you think?

Reply:
Yes, sounds good